internal/binary/envoy/debug/node.go

```diff
@@ -187,7 +187,7 @@ func printProcessTable(out io.Writer, parsed []*proc) error {
 func (n *nodeCollection) networkInterfaces() error {
 	result, err := net.Interfaces()
 	if err != nil {
-		return fmt.Errorf("unable to fetch network Interfaces: %w", err)
+		return fmt.Errorf("unable to read network Interfaces: %w", err)
```

Comment:
I think `list` would be more precise in this context: https://github.com/golang/go/blob/master/src/net/interface.go#L99

Reply:
`read` makes me feel like I'm reading frames from NIC literally, nit 😄

Reply:
def better